### PR TITLE
Kafka producer metric

### PR DESCRIPTION
**Pull request by dragutin-nav**

### CODE REVIEW

File: src/main/java/no/nav/veilarboppfolging/repository/KafkaProducerRepository.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public class KafkaProducerRepository {
+    private final JdbcTemplate db;
+
+    @Autowired
+    public KafkaProducerRepository(JdbcTemplate db) {
+        this.db = db;
+    }
+
+    public Integer getNumberOfUnprocessedMessages() {
+        return db.queryForObject("SELECT COUNT(*) FROM KAFKA_PRODUCER_RECORD WHERE CREATED_AT < SYSDATE - INTERVAL '30' MINUTE",
```
Comment (stianStensli) [inline src/main/java/no/nav/veilarboppfolging/repository/KafkaProducerRepository.java:17]: Maybe we can return the oldest CREATED_AT date? or null or now() if there are no messages, and then grafana can trigger the alert if the date is too old?

Comment (stianStensli) [reply src/main/java/no/nav/veilarboppfolging/repository/KafkaProducerRepository.java:17]: or maybe return now() - "oldest CREATED_AT" to get how old the oldest message is?

Comment (dragutin-nav) [reply src/main/java/no/nav/veilarboppfolging/repository/KafkaProducerRepository.java:17]: Aha, yep, that can be useful as well

File: src/main/java/no/nav/veilarboppfolging/repository/KafkaProducerRepository.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+public class KafkaProducerRepository {
+    private final JdbcTemplate db;
+
+    @Autowired
+    public KafkaProducerRepository(JdbcTemplate db) {
+        this.db = db;
+    }
+
+    public Integer getOldestMessage() {
+        return db.queryForObject("SELECT MAX(ROUND(((SYSDATE+0)-(OPPDATERT+0)) * 24 * 60)) as minutes_processed_ago FROM KAFKA_PRODUCER_RECORD",
```
Comment (stianStensli) [inline src/main/java/no/nav/veilarboppfolging/repository/KafkaProducerRepository.java:17]: Does this handle OPPDATERT == null?

**Review by matiasvinjevoll — APPROVED** [review]

**Pull request merged**